### ISSUE TITLE
UUID in Settings not correctly formatted

### ISSUE THREAD

**Issue by pe224** [Pushjet/Pushjet-Android#26]
Great project!
The UUID for the device is not correctly formatted.
It seems to convert the UUID to some kind of float (see screenshot below).
![image](https://user-images.githubusercontent.com/8567333/29418631-36bca498-836d-11e7-926c-d5b6fefd8f0d.png)

**Comment by Mechazawa** [comment]
Yeah this is only a cosmetic issue for what I can recall. If I'm not mistake when you change it it will be properly formatted etc. I haven't used my phone (or any) in a while so this is just going from memory.

**Comment by z3tof** [comment]
I've just find this project !
Great project ! Still alive project ?

How can I find my UUID ?

Even when I change it, it is not properly formatted... :-(

**Comment by mrtnmtth** [comment]
The project is unmaintained but the servers are up and running and everything should be working.

The option is only for setting a custom ID.
There is no build-in way to display your device's UUID.
What do you need it for? There should not be any reason for you to need to know it.

If you would like to know how the UUID is generated, have a look at [DeviceUuidFactory.java](https://github.com/Pushjet/Pushjet-Android/blob/master/app/src/main/java/io/Pushjet/api/PushjetApi/DeviceUuidFactory.java).

**Comment by z3tof** [comment]
Thanks for your answer.

I though that I need my UUID, but finally, I don't need it (I didn't understand how it worked)

Indeed, everything works fine... (the bug of the incorrect time zone is however annoying).

It's a great project... too bad it's no longer being maintained...

**Comment by ghost** [comment]
Hi

I try to use pushjet as a persistant notification for my homemade home automation system.

I'm looking for my UUID in order to be able to delete old message from my system and replace it by the new informations
Any idea on how to get it (maybe @mttronc ? )

**Comment by mrtnmtth** [comment]
Took me a while to figure out what you are trying to accomplish. As far as I get it, you want to send a DELETE request to the messages API.

The GET request returns all unfetched messages as JSON and removes them from the server database. The DELETE request just removes them.  
So all received messages are already deleted from the server.

For normal use cases you won't ever need to issue a DELETE request.
